redbot/cogs/mod/mod.py

```diff
@@ -1304,36 +1304,50 @@ def __unload(self):
 
         if not user:
             user = author
+        if isinstance(user, int):
+            user_id = user
+            user = await self.bot.get_user_info(user_id)
```

Comment:
`get_user_info` isn't like `get_member` and the like - if the user ID is not found, it raises a 404 Not Found error (specifically `discord.NotFound`) instead of returning `None`.

It's also worth noting that there's a `get_user` method of `bot`, which will fetch the user from the bot's global cache of connected users. It may be worth checking that before using `get_user_info`.